help/blocks/accordion/accordion.js

```diff
@@ -36,50 +38,64 @@ function showView(block, subcategoryUrls, categoryTitle) {
 
   const accordionSectionSubCategoryDiv = document.createElement('div');
   accordionSectionSubCategoryDiv.classList.add('accordion-subcategory-div');
-  subcategoryUrls.forEach((item) => {
-    const sectionAnchor = document.createElement('div');
-    sectionAnchor.classList.add('content');
-    sectionAnchor.innerHTML = `
+  if (!isVariation) {
+    accordionBody.forEach((item) => {
+      const sectionAnchor = document.createElement('div');
+      sectionAnchor.classList.add('content');
+      sectionAnchor.innerHTML = `
         <a href='${item.path}'>
             <span>${item.shorttitle}</span>
         </a>
-      `;
-    accordionSectionSubCategoryDiv.appendChild(sectionAnchor);
-  });
+        `;
+      accordionSectionSubCategoryDiv.appendChild(sectionAnchor);
+    });
+  } else {
+    accordionBody.classList.add('content');
+    accordionSectionSubCategoryDiv.appendChild(accordionBody);
+  }
   accordionSectionSubCategoryDiv.setAttribute('aria-hidden', 'true');
   accordionSection.appendChild(accordionSectionSubCategoryDiv);
   block.appendChild(accordionSection);
 }
 
 export default async function decorate(block) {
-  const pageIndex = window.siteindex.data;
-  const indexPath = window.location.pathname;
-  const cat = getMetadata('category');
-  const subcat = getMetadata('subcategory');
+  isVariation = document.querySelector('.content-center') !== null;
```

Comment:
What is the most used variation ? Centered or the other one ?

Reply:
Most used is the other one - in category and subcategory pages.
While centered one is used in some articles as content.

Reply:
Ok - wanted to make sure that authors add variations for least used cases.